tools/update_versions.py

```diff
@@ -67,6 +67,20 @@ def _get_git_log(repo, since_id):
 
     return log
 
+def _get_git_last_commit_log(repo, last_id):
+    """
+    Get log of short SHA1 and short message for last commit
+    """
+    log = []
+    for commit in repo.iter_commits():
```

Comment:
Use `repo.commit` instead of iterating.

https://gitpython.readthedocs.io/en/stable/tutorial.html#the-commit-object